src/main/java/com/google/sps/servlets/AvailabilityServlet.java

```diff
@@ -28,10 +28,32 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.sps.utilities.RealAvailabilityDatastore;
+import com.google.sps.utilities.MockAvailabilityDatastore;
+import com.google.sps.utilities.AvailabilityDatastoreService;
 
+/** Servlet that getting and adding availability for a tutor. */
 @WebServlet("/availability")
 public class AvailabilityServlet extends HttpServlet {
 
+    private AvailabilityDatastoreService datastore;
+
+    /**
+    * Because we created a constructor with a parameter (the testing one), the default empty constructor does not work anymore so we have to explicitly create it. 
+    * We need the default one for deployment because the servlet is created without parameters.
+    */
+    public AvailabilityServlet(){}
+
+    public AvailabilityServlet(boolean test) {
```

Comment:
Would we need to add something like this to every servlet that gets the availability of a tutor so that we can test properly?

Reply:
Yes, we would need to add a new constructor like this to every servlet that uses a datastore service class (not just getting availability).